marklogic-data-hub-central/ui/src/components/header/header.tsx

```diff
@@ -141,25 +215,54 @@ const Header:React.FC<Props> = (props) => {
         theme="dark"
       >
         <Menu.Item>
-          <MLTooltip title="Help"><a id="help-link" href="https://docs.marklogic.com/datahub/" target="_blank"><Icon type="question-circle"/></a></MLTooltip>
+          <MLTooltip title="Help">
+            <div className={styles.helpIconContainer}>
+              <a id="help-link" href="https://docs.marklogic.com/datahub/" target="_blank" tabIndex={1} className={styles.helpIconLink}>
+                <Icon type="question-circle" className={styles.helpIcon}/>
+              </a>
+            </div>
+          </MLTooltip>
         </Menu.Item>
       </Menu>
     </div>;
   }
 
-  const handleHomeClick = () => {
+  const handleHomeClick = (event) => {
+    event.preventDefault();
     props.history.push("/tiles");
   };
 
+  const logoKeyDownHandler = (event) => {
+    if (event.key === "ArrowRight") { titleRef.current!.focus(); }
+  };
+
+  const logoClickHandler = (event) => {
+    event.preventDefault();
+    logoRef.current!.click();
+  };
+
+  const titleKeyDownHandle = (event) => {
+    if (event.key === "Enter") { handleHomeClick(event); }
+
+    if (event.key === "ArrowRight") { serviceNameRef.current!.focus(); }
+    if (event.key === "ArrowLeft") { logoRef.current!.focus(); }
+  };
+
   return (
     <>
       <Layout.Header className={styles.container}>
-        <div className={styles.logoContainer} aria-label="header-logo" onClick={handleHomeClick}>
-          <Avatar className={styles.logo} src={logo} />
+        <div className={styles.logoContainer} aria-label="header-logo">
+          <a href="https://www.marklogic.com/" aria-label="logo-link" className={styles.logo} tabIndex={1} ref={logoRef} 
```

Comment:
We've updated the header logo to link to marklogic.com. There's a test in App.test.tsx (line 46) that checks that this goes to the Overview page when clicked. I think this needs to be updated to test the new link, it's failing in my env.

Reply:
fixed